### PR TITLE
Update pyparsing to 2.1.10

### PR DESCRIPTION
**Pull request by tino097**
Updating pyparsing to 2.1.10 to fix  build setup

### CODE REVIEW

File: ckanext/spatial/geoalchemy_common.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 def postgis_version():
 
-    result = Session.execute('SELECT postgis_lib_version()')
+    result = Session.execute('SELECT PostGIS_Lib_Version()')
```
Comment (torfsen) [inline ckanext/spatial/geoalchemy_common.py]: How is this related to pyparsing?

Comment (tino097) [inline ckanext/spatial/geoalchemy_common.py]: that file shouldn't have been with the pull request, my bad, i will remove that commit

Comment (tino097) [inline ckanext/spatial/geoalchemy_common.py]: there is new PR #173
